include/z64save.h

```diff
@@ -888,28 +888,32 @@ typedef enum {
 // Cremia did Milk Run alone. Player didn't interact or didn't accept the ride
 #define WEEKEVENTREG_34_80 PACK_WEEKEVENTREG_FLAG(34, 0x80)
 
-#define WEEKEVENTREG_35_01 PACK_WEEKEVENTREG_FLAG(35, 0x01)
-#define WEEKEVENTREG_35_02 PACK_WEEKEVENTREG_FLAG(35, 0x02)
-#define WEEKEVENTREG_35_04 PACK_WEEKEVENTREG_FLAG(35, 0x04)
-#define WEEKEVENTREG_35_08 PACK_WEEKEVENTREG_FLAG(35, 0x08)
-#define WEEKEVENTREG_35_10 PACK_WEEKEVENTREG_FLAG(35, 0x10)
-#define WEEKEVENTREG_35_20 PACK_WEEKEVENTREG_FLAG(35, 0x20)
+// Bought each possible map from Tingle
+#define WEEKEVENTREG_BOUGHT_TINGLE_MAP_CLOCK_TOWN PACK_WEEKEVENTREG_FLAG(35, 0x01)
```

Comment:
```suggestion
#define WEEKEVENTREG_TINGLE_MAP_BOUGHT_CLOCK_TOWN PACK_WEEKEVENTREG_FLAG(35, 0x01)
```
Kind of nitpicky, but I think I'd prefer `TINGLE_MAP` coming before `BOUGHT` making `TINGLE_MAP`  like a system.